hugolib/pagination.go

```diff
@@ -414,6 +418,39 @@ func paginatePages(td targetPathDescriptor, seq interface{}, pagerSize int) (pag
 	return pagers, nil
 }
 
+func toPagesGroup(seq interface{}) (PagesGroup, error) {
+	if seq == nil {
+		return nil, nil
+	}
+
+	switch seq.(type) {
+	case PagesGroup:
+		return seq.(PagesGroup), nil
+	case []PageGroup:
+		return PagesGroup(seq.([]PageGroup)), nil
+	case []interface{}:
+		list := seq.([]interface{})
+		l := len(list)
+		if l == 0 {
+			break
+		}
+		switch list[0].(type) {
+		case PageGroup:
+			pagesGroup := make([]PageGroup, len(list))
```

Comment:
` make(PagesGroup, len(list))` is better.

Reply:
Done